dateparser/parser.py

```diff
@@ -346,16 +346,20 @@ def _get_date_obj(self, token, directive):
         return strptime(token, directive)
 
     def _results(self):
-        if self.settings.STRICT_PARSING:
-            errors = []
-            if not self.day:
-                errors.append('Day')
-            if not self.month:
-                errors.append('Month')
-            if not self.year:
-                errors.append('Year')
+        missing = []
+        if not self.day:
+            missing.append('day')
+        if not self.month:
+            missing.append('month')
+        if not self.year:
+            missing.append('year')
```

Comment:
:lipstick: 

```suggestion
        missing = [field for field in ('day, 'month', 'year')
                   if not getattr(self, field)]
```

Reply:
Beautiful! :heart:

Reply:
There is a single quote (`'`) missing after `day`, but don't worry, I added it :smile: